Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1349,3 +1354,63 @@ private class BrowserScreenshotHelper: ScreenshotHelper {
         return nil
     }
 }
+
+private class BrowserPrintPageRenderer: UIPrintPageRenderer {
```

Comment:
Move this to its own class (unless it has dependencies it needs in here for some reason, I prefer the isolation)